### PR TITLE
Init

### PR DESCRIPTION
**Pull request by liss-h**

### CODE REVIEW

File: generate_update_operations.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```
Comment (bigerl) [inline generate_update_operations.sh:1]: I would really appreciate if we could do this in a modern, expressive language. Python, Kotlin, rust or even JS would be fine.

File: generate_update_operations.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            insert_template=""
+            for ((y=0;y<$template_size;y++));
+            do
+                insert_template="${insert_template}${current_subject} ${INSERT_PREDICATE} \"${INSERT_INTEGER_LITERAL}\" . "
+                INSERT_INTEGER_LITERAL=$((INSERT_INTEGER_LITERAL + 1))
+            done
+            op_str="INSERT DATA { $insert_template }"
+            echo $op_str > "$worker_dir/op_$j.ru"
+        fi
+        curl -X POST "$UPDATE_ENDPOINT_URL" -H 'Content-Type: application/sparql-update' -d "$op_str"
```
Comment (bigerl) [inline generate_update_operations.sh:67]: If I get this right, you don't manually calculate the expected result but use another triplestore (like fuseki) as reference.

**Review by bigerl — CHANGES_REQUESTED** [review]
I'll leave this one here already. For the rest, I am still busy reviewing.

File: src/update_worker.rs [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        }
+
+        Ok(Self { id, query_endpoint, update_endpoint, client: Client::new(), queries })
+    }
+
+    async fn read_current_state(&self, subject: &str) -> anyhow::Result<DbState> {
+        let state = self
+            .client
+            .get(self.query_endpoint.clone())
+            .query(&[("query", Self::make_verify_query(subject))])
```
Comment (bigerl) [inline src/update_worker.rs]: th Accept header should be set to "application/n-triples" to ensure that the endpoint delivers the expected format.

**Review by bigerl — CHANGES_REQUESTED** [review]
Only a little nit-pick that is ATM not really relevant for Tentris. Otherwise, the rust code looks good.

File: generate_update_operations.sh (hunk 1 above)
Comment (liss-h) [reply generate_update_operations.sh:1]: Will create an issue

**Pull request merged**